src/Instrumentation/Laravel/composer.json

```diff
@@ -12,17 +12,15 @@
     "ext-json": "*",
     "ext-opentelemetry": "*",
     "laravel/framework": ">=6.0",
-    "open-telemetry/api": "^1.0.0beta10",
+    "open-telemetry/api": "^1.0",
+    "open-telemetry/sdk": "^1.0",
```

Comment:
Is there a way for it to not directly use the SDK? The OTEL spec is fairly firm about it: https://opentelemetry.io/docs/specs/otel/library-guidelines/#requirements

Reply:
Thanks for pointing it out, I was not aware. Given that rule, it looks like the current implementation might be best in not using the ClockFactory, if possible?

I shall take a look at removing that coupling.

Reply:
@brettmc I've made changes to decouple from the SDK implementation here: https://github.com/open-telemetry/opentelemetry-php-contrib/pull/211/commits/96038b7606e6c07acb7e2f78f949d5671c50fb88

Reply:
I'm happy with your changes. We could maybe move `ClockInterface` into the API (since it has some constants that might make that code a little more readable), but it's a pretty minor use-case and nothing else in contrib needs if, AFAICT.